src/debugtopus/index.ts

```diff
@@ -260,3 +260,84 @@ export const runDebugtopus = async (options: DebugtopusOptions) => {
   });
   await runTests({ ...dirs, runMode: options.headless ? "headless" : "ui" });
 };
+
+export const executeLocalTestCases = async (
+  options: DebugtopusOptions & { source: string },
+): Promise<void> => {
+  const testCases = readTestCasesFromDir(options.source);
+  const body = {
+    testCases,
+    testTargetId: options.testTargetId,
+    executionUrl: options.url,
+    environmentId: options.environmentId,
+  };
+  const { error, response } = await client.GET(
+    "/apiKey/beta/test-targets/{testTargetId}/code",
+    {
+      params: {
+        path: {
+          testTargetId: options.testTargetId,
+        },
+      },
+      body,
+      parseAs: "stream",
+    },
+  );
+
+  if (error || !response?.body) {
+    handleError(error);
+  }
+
+  const dirs = await prepareDirectories();
```

Comment:
can we split this code and below into a separate method? (and write some tests 😬 )